pkg/templates/targetport/internal/params/params.go

```diff
@@ -0,0 +1,7 @@
+package params
+
+// Params represents the params accepted by this template.
+type Params struct {
+	// Set to true to validate targetPort names
+	TargetPort bool
```

Comment:
I don't think we need this param. This seems like a param that controls whether the check is enabled or not, which is not required, since enabling/disabling the check is controlled at a higher level. This check can be param-less.

Reply:
✅